cmd/avrogo/internal/generated_tests/justEnum/roundtrip_test.go

```diff
@@ -0,0 +1,31 @@
+// Code generated by generatetestcode.go; DO NOT EDIT.
+
+package justEnum
+
+import (
+	"testing"
+
+	"github.com/heetch/avro/cmd/avrogo/internal/testutil"
+)
+
+var tests = testutil.RoundTripTest{
+	InSchema: `{
```

Comment:
Just out of curiosity.
It's not easy to understand how these tests work.

Why do we need to define a schema.avsc and the InSchema?
They are equal

Reply:
This code is generated from CUE tests, that's the actual changes I did was adding a test. This test makes a roundtrip test:
1. Parse schema
2. Encode input data as binary with generated code
3. Decode binary and make sure it's equal to original input data
That's done for all generated tests.